packages/marginfi-client-v2/src/bank.ts

```diff
@@ -333,7 +328,17 @@ export interface BankData {
 
 export enum OracleSetup {
   None = 0,
-  Pyth = 1,
+  PythEma = 1,
+  SwitchboardV2 = 2
+}
+
+export function decodeOracleSetup(oracleSetup: object): OracleSetup {
+  switch (Object.keys(oracleSetup)[0]) {
+    case "pyth": return OracleSetup.PythEma;
```

Comment:
do we need `pyth` and `pythema` here?

Reply:
for some reason its still being mapped to `{ 'pyth': {} }` despite me updating the IDL, will keep for backwards compatibility while I debug later. don't want to get stuck on this now.